tests/servers/tests.py

```diff
@@ -54,17 +53,21 @@ def test_live_server_url_is_class_property(self):
 class LiveServerViews(LiveServerBase):
     def test_protocol(self):
         """Launched server serves with HTTP 1.1."""
-        with captured_stdout() as debug_output:
-            conn = HTTPConnection(LiveServerViews.server_thread.host, LiveServerViews.server_thread.port)
-            try:
-                conn.set_debuglevel(1)
-                conn.request('GET', '/example_view/', headers={"Connection": "keep-alive"})
-                conn.getresponse().read()
-                conn.request('GET', '/example_view/', headers={"Connection": "close"})
+        with self.urlopen('/example_view/') as f:
+            self.assertEqual(f.version, 11)
+
+    @override_settings(MIDDLEWARE=[])
+    def test_closes_connection(self):
+        conn = HTTPConnection(LiveServerViews.server_thread.host, LiveServerViews.server_thread.port, timeout=1)
```

Comment:
I'd add a docstring for `test_closes_connection` because the purpose of that test may not be obvious (e.g. why `MIDDLEWARE=[]`?).